commands/save_all.go

```diff
@@ -0,0 +1,36 @@
+// This is free and unencumbered software released into the public
+// domain.  For more information, see <http://unlicense.org> or the
+// accompanying UNLICENSE file.
+
+package commands
+
+import (
+	"github.com/nelsam/gxui"
+)
+
+type AllSaver interface {
+	SaveAll()
+}
+
+type SaveAll struct{}
+
+func NewSaveAll(theme gxui.Theme) *SaveAll {
+	return &SaveAll{}
+}
+
+func (s *SaveAll) Name() string {
+	return "save-all-files"
+}
+
+func (s *SaveAll) Menu() string {
+	return "File"
+}
+
+func (s *SaveAll) Exec(target interface{}) (executed, consume bool) {
+	saver, ok := target.(EditorWithSave)
```

Comment:
We're still type asserting to `EditorWithSave`, here.  Does this compile?  I think it needs to assert to `AllSaver`, right?